From f87858439fe0a6190a6ed5792b6e20ac5253bff9 Mon Sep 17 00:00:00 2001
From: Quanyang Wang <quanyang.wang@windriver.com>
Date: Wed, 10 Apr 2019 13:48:07 +0800
Subject: [PATCH 16/30] mxc: sim: imx_emvsim: include header file explicitly to
 avoid compile error
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

This patch fixes compiling error as below:
drivers/mxc/sim/imx_emvsim.c: In function ‘emvsim_suspend’:
drivers/mxc/sim/imx_emvsim.c:1587:2: error: implicit declaration of function ‘pinctrl_pm_select_sleep_state’ [-Werror=implicit-function-declaration]
  pinctrl_pm_select_sleep_state(&pdev->dev);
  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
drivers/mxc/sim/imx_emvsim.c: In function ‘emvsim_resume’:
drivers/mxc/sim/imx_emvsim.c:1601:2: error: implicit declaration of function ‘pinctrl_pm_select_default_state’ [-Werror=implicit-function-declaration]
  pinctrl_pm_select_default_state(&pdev->dev);
  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
cc1: some warnings being treated as errors

Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
---
 drivers/mxc/sim/imx_emvsim.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/mxc/sim/imx_emvsim.c b/drivers/mxc/sim/imx_emvsim.c
index 59f54dcd533e..3b6eb75aeb5d 100644
--- a/drivers/mxc/sim/imx_emvsim.c
+++ b/drivers/mxc/sim/imx_emvsim.c
@@ -29,6 +29,7 @@
 #include <linux/spinlock.h>
 #include <linux/time.h>
 #include <linux/types.h>
+#include <linux/pinctrl/consumer.h>
 
 #define	DRIVER_NAME	"mxc_emvsim"
 
-- 
2.17.1

